docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -6,14 +6,14 @@ approvers:
 title: Kubernetes Cloud Controller Manager
 ---
 
-**Cloud Controller Manager is an alpha feature in 1.8. In upcoming releases it will be the preferred way to integrate Kubernetes with any cloud. This will ensure cloud providers can develop their features independently from the core Kubernetes release cycles.**
+**Cloud Controller Manager is an alpha feature in 1.9. In upcoming releases it will be the preferred way to integrate Kubernetes with any cloud. This will ensure cloud providers can develop their features independently from the core Kubernetes release cycles.**
 
 * TOC
 {:toc}
 
 ## Cloud Controller Manager
 
-Kubernetes v1.6 contains a new binary called `cloud-controller-manager`. `cloud-controller-manager` is a daemon that embeds cloud-specific control loops. These cloud-specific control loops were originally in the `kube-controller-manager`. Since cloud providers develop and release at a different pace compared to the Kubernetes project, abstracting the provider-specific code to the `cloud-controller-manager` binary allows cloud vendors to evolve independently from the core Kubernetes code.
+Kubernetes v1.6 introduced a new binary called `cloud-controller-manager`. `cloud-controller-manager` is a daemon that embeds cloud-specific control loops. These cloud-specific control loops were originally in the `kube-controller-manager`. Since cloud providers develop and release at a different pace compared to the Kubernetes project, abstracting the provider-specific code to the `cloud-controller-manager` binary allows cloud vendors to evolve independently from the core Kubernetes code.
```

Comment:
This, is confusing. If the binary was introduced in v1.6, why it is labelled an alpha only in v1.9? Please clarify.

Reply:
@tengqm This is my first time reading the k8s docs so I'm confused as well 😛. Would you expect `cloud-controller-manager` to no longer be alpha? The [feature docs](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/) says it's alpha in 1.8, so I assume it's still alpha in 1.9.

Reply:
@mattayes Need to check the upstream source to find if it is still alpha, or it has been upgraded to a beta. Better find someone familiar with its history for a quick suggestion.

Reply:
@tengqm It looks like `cloud-controller-manager` is still in beta: kubernetes/kubernetes#48690.

Reply:
@mattayes thanks for this - since this is still an alpha feature, would you mind annotating this page with `{% include feature-state-alpha.md %}` - from https://kubernetes.io/docs/home/contribute/includes/#alpha-feature, for clarity?

Reply:
@heckj On it!

Reply:
@heckj Fixed.